### ISSUE TITLE
IncrementalBar has unprintable characters on Windows on Python 3.6

### ISSUE THREAD

**Issue by zooba** [verigak/progress#36]
Python 3.6 has updated the console on Windows to print Unicode characters correctly and always use UTF-8 for `sys.std[in|out|err].encoding`. Unfortunately, 6 of the characters in `IncrementalBar.phases` do not have glyphs in the default console fonts, so the bar displays the missing glyph image (question mark in a box) while incrementing.

Programs (like `pip`) that check `stdout.encoding` is Unicode before using `IncrementalBar` will now use it instead of the non-incremental variety. I made the tweak below in my copy of pip and the result was acceptable, if not especially "incremental" anymore. Considering the extra glyphs are probably not going to be added anytime soon, and are unlikely to make it onto existing versions of Windows anyway, I'd like to propose this change (with appropriate comments as you wish) to avoid an ugly bar.

```python
class IncrementalBar(Bar):
    if sys.platform.startswith('win'):
        phases = (u' ', u'▌', u'█')
    else:
        phases = (u' ', u'▏', u'▎', u'▍', u'▌', u'▋', u'▊', u'▉', u'█')
```

**Comment by verigak** [comment]
Thanks for the ping!

**Comment by pradyunsg** [comment]
Thanks for the fix! :)

Do you have a timeline in mind for when you'll be releasing a version with this fix?

**Comment by verigak** [comment]
There's not a lot of stuff comitted since the last release but I could do a new one if nothing breaks. I'll wait at least a couple weeks to make sure nobody complains and if you are still interested then, you can ping me again to do a release.

**Comment by pradyunsg** [comment]
Sure thing. No hurries. Thanks! :)

**Comment by pradyunsg** [comment]
Hey @verigak!

Are you fine doing a new release now?

**Comment by verigak** [comment]
Released v1.4

**Comment by benshep** [comment]
I can see these characters in Windows now (OK in PyCharm and Windows Terminal at least in Win11). Could the extra phases be re-enabled? Or maybe include a 'legacy mode' for IncrementalBar for older consoles that has the limited characters? I really like the smooth-updating IncrementalBar
